src/sentry/monitors/consumers/check_in.py

```diff
@@ -83,6 +87,16 @@ def _process_message(wrapper: Dict) -> None:
 
     project = Project.objects.get_from_cache(id=project_id)
 
+    ratelimit_key = params["monitor_slug"]
+
+    if ratelimits.is_limited(
+        f"monitor-checkins:{ratelimit_key}",
+        limit=CHECKIN_QUOTA_LIMIT,
+        window=CHECKIN_QUOTA_WINDOW,
+    ):
```

Comment:
We should increment `monitors.checkin.dropped.ratelimited` here too. Tbh, we should probably centralize the rate limiting code.

I think it might also be worth tagging any stats in the api vs relay in case we need to break them down

Reply:
Yeah I agree with that. Getting a bit late in the day to get this in, so I'll wait until monday

Reply:
done